scripts/gendump/main.go

```diff
@@ -99,7 +99,7 @@ func main() {
 			emit.AppCall(w.BinWriter, contractHash, "put", callflag.All, key, value)
 			handleError("can't create transaction", w.Err)
 
-			tx := transaction.New(w.Bytes(), 4_000_000)
+			tx := transaction.New(w.Bytes(), 4_040_000)
```

Comment:
Is this a fix by itself?

Reply:
Yes, we haven't updated it quite a long time and the prices have been changed. Try to run 
```
go run ./scripts/gendump/main.go --out ./cli/testdata/chain50x2.acc --blocks 50 --txs 2
```
at the current master and it will fail.